examples/plot_cross_subject_ssvep.py

```diff
@@ -0,0 +1,209 @@
+"""
+===========================
+Cross Subject SSVEP
+===========================
+This example shows how to perform a cross subject analysis on a SSVEP dataset.
+We will compare two pipelines :
+- Riemannian Geometry
+- CCA
+We will use the SSVEP paradigm, which uses the AUC as metric.
+"""
+# Authors: Sylvain Chevallier <sylvain.chevallier@uvsq.fr>
+#
+# License: BSD (3-clause)
+
+from sklearn.pipeline import make_pipeline
+from sklearn.cross_decomposition import CCA
+from sklearn.linear_model import LogisticRegression
+from sklearn.base import BaseEstimator, TransformerMixin, ClassifierMixin
+
+from pyriemann.tangentspace import TangentSpace
+from pyriemann.estimation import Covariances
+
+from moabb.evaluations import CrossSubjectEvaluation
+from moabb.paradigms import SSVEP, FilterBankSSVEP
+from moabb.datasets import SSVEPExo
+import moabb
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import warnings
+
+warnings.simplefilter(action='ignore', category=FutureWarning)
+warnings.simplefilter(action='ignore', category=RuntimeWarning)
+moabb.set_log_level('info')
+
+###############################################################################
+# Auxiliary functions
+# -------------------
+#
+# We will need an auxiliary transformer for filtering the signal.
+# This auxiliary transformer allows to reshape the signal for building
+# extended covariances matrices
+
+
+class FilteredSignal(BaseEstimator, TransformerMixin):
```

Comment:
I think a more natural place for this would be in `moabb/pipeline`

Reply:
Right, it is moved.